internal/providers/terraform/aws/testdata/global_accelerator_test/global_accelerator_test.golden

```diff
@@ -0,0 +1,15 @@
+
+ Name                                                                    Monthly Qty  Unit   Monthly Cost 
+                                                                                                          
+ TestFixedFee                                                                                             
+ └─ AWS Global Accelerator TestFixedFee Fixed Fee                                730  hours        $18.25 
+                                                                                                          
+ TestUsage                                                                                                
+ ├─ AWS Global Accelerator TestUsage Fixed Fee                                   730  hours        $18.25 
+ ├─ AWS Global Accelerator TestUsage DT-Premium Usage OUT from NA to AP        1,000  GB           $35.00 
```

Comment:
As noted below, this should move to a `aws_globalaccelerator_endpoint_group` and follow the [cost component naming conventions](https://github.com/infracost/infracost/blob/master/contributing/resource_mapping_guide.md#cost-component-names-and-units).  To be consistent with the cloud front distributions, this should look something like:
```suggestion
    ├─ DT-Premium Outbound to Asia Pacific 
    ├─ DT-Premium Outbound to Europe & Israel 
```